frontend/src/components/module-editor/BlockPreview.tsx

```diff
@@ -13,14 +20,48 @@ const BlockStyle = ({
   className?: string;
   content: ContentTypeEnum;
 }): React.ReactElement => {
+  let contentIcon: React.ReactElement = <ImageIcon />;
+  // eslint-disable-next-line default-case
```

Comment:
I know you likely won't be hitting the default case on an enum-based switch statement, but it's probably best to avoid ignoring the eslint error here and adding a default case.